### ISSUE TITLE
`libGL.so.1 not found` on centos 64bit

### ISSUE THREAD

**Issue by chemdemo** [paralect/robomongo#688]
input:

``` bash
cd bin
chmod u+x robomongo.sh
./robomongo.sh
```

output:

``` bash
/usr/local/lib/robomongo-0.8.4-x86_64/bin/robomongo: error while loading shared libraries: libGL.so.1: cannot open shared object file: No such file or directory
```

**Comment by shanthik** [comment]
I tried installing roboMongo on my linux machine but on executing ./robomongo.sh it gives the following error -
error while loading shared libraries: libGL.so.1: cannot open shared object file: No such file or directory.

What is the issue here could anyone help ?

**Comment by stennie** [comment]
@shanthik It's a packaging issue with 0.8.4 RPM. Will be fixed in the 0.8.5 packages (I'll post a preview rpm to #783 when it's ready).

As a workaround, I expect you can fix the dependency with `sudo yum install mesa-libGL` but I'm not sure if that's the only missing .so.
